src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -302,11 +302,6 @@ function isCtrlKey(e) {
 })(document);
 </script>
 <style>
-pre.sf-dump {
-    display: block;
-    white-space: pre;
-    padding: 5px;
-}
```

Comment:
Hmm, I see, but these are actually overwritable by the `default` style. For instance, `white-space: pre` is always overwritten by `white-space: pre-wrap` from the `default` style. Maybe #13487 should have changed this in the block directly.

Anyway, I'll revert those changes, as this is not related to the PR.